src/main/scala/edu/cmu/cs/obsidian/parser/AST.scala

```diff
@@ -100,7 +100,11 @@ case class VariableDecl(typ: ObsidianType, varName: String) extends Statement
 case class VariableDeclWithInit(typ: ObsidianType, varName: String, e: Expression) extends Statement
 
 case class VariableDeclWithSpec(typIn: ObsidianType, typOut: ObsidianType, varName: String) extends Statement
-
+ {
+     override def toString: String = {
```

Comment:
This could just be `override def toString: String = varName`

Reply:
OK, sure.